__tests__/cmds/openapi.test.js

```diff
@@ -0,0 +1,226 @@
+const nock = require('nock');
```

Comment:
Looks like this is the full content of `swagger.test.js` so since the command functionality is the same we should be able to rename that to this instead and then add a test to make sure that hitting `rdme swagger` loads the `rdme openapi` command.

Reply:
Okay perfect, I wasn't sure if we were trying to test to make sure they worked the same. I'll change that, thanks!